src/resolvers/mutations/subject.js

```diff
@@ -75,6 +75,17 @@ const addSubject = async (_, { input }, context, _info) => {
             return utils.sendErrorResponse(field, message);
         }
 
+        if (input['department']) {
+            message = 'Department does not exist';
+            field = 'department';
+            const departments = await context.prisma.school({ id: school }).department();
+
+            const depExists = departments.filter(item => item?.id == input['department']['connect']['id']);
+            if (!depExists.length) {
+                return utils.sendErrorResponse(field, message);
+            }
+        }
+
         message = 'Subject already exists in this school';
```

Comment:
Can you add this validation for subjectAbv and subject code too.

Reply:
Sure